flow-typed/config.js

```diff
@@ -14,6 +14,9 @@ declare type JSONFormatConfig = {|
   +size: number,
 |}
 
+type BlockExplorerAPILink = 'https://api-blockchain-service.jwallet.network' |
+  'http://api-blockchain-service.jwallet.develop.stacks.jdev.network'
```

Comment:
It is our own internal development infrastructure, I think it is not good to leave it in opensource.
Let's discuss it offline

Reply:
It is already in opensource. ;)

Reply:
I think, we should move these values to environment variables so they could be set in run-time. Like those `[MAIN_RPC_ADDR]` in index.html

Reply:
But in types it should be probably just string. There is no need for enum with configuration variables, it feels a little bit off...

Reply:
At least for localhost dev we need default value without any params from docker image

Reply:
Yep, but we can pass env variables through webpack, without using Docker

Reply:
See here: https://github.com/jibrelnetwork/jwallet-web/blob/develop/webpack.config.js#L266-L271